borg/archiver.py

```diff
@@ -1399,7 +1440,7 @@ def build_parser(self, args=None, prog=None):
         subparser.add_argument('location', metavar='ARCHIVE',
                                type=location_validator(archive=True),
                                help='name of archive to create (must be also a valid directory name)')
-        subparser.add_argument('paths', metavar='PATH', nargs='+', type=str,
+        subparser.add_argument('paths', metavar='PATH', nargs='*', type=str,
```

Comment:
ok, so not really a great workaround. i guess some people will want to have all in a separate file, so they can use same script on their different hosts - and deal with all roots, all excludes/includes in the patterns file. and still not being unnerved by warnings and rc==1 all the time.

so, just ignore the argparse bug and add a ticket or faq item about it?